cmake/Hunter/config.cmake

```diff
@@ -1,39 +1,37 @@
+# FIXME, waiting for release:
+# * https://github.com/jarro2783/cxxopts/issues/110#issuecomment-394909564
+hunter_config(
+    cxxopts
+    VERSION 2.1.1
+    URL https://github.com/jarro2783/cxxopts/archive/e725ea308468ab50751ba7f930842a4c061226e9.zip
+    SHA1 cbeec5576599d031f6f992d987e1f3575b3afee3
+)
+
```

Comment:
This will be used for the `LOCAL` builds, while CI builds will still use https://github.com/elucideye/drishti-upload/blob/master/config.cmake.

The cxxopts update will probably close https://github.com/elucideye/drishti/issues/593

Reply:
`drishti-upload` updated